src/towncrier/_git.py

```diff
@@ -16,11 +28,23 @@ def remove_files(fragment_filenames, answer_yes):
     else:
         click.echo("I want to remove the following files:")
 
-    for filename in fragment_filenames:
+    for filename in sorted(fragment_filenames):
         click.echo(filename)
 
+    # Filter out files that are unknown to git
+    try:
+        known_fragments = run(
```

Comment:
Maybe rename to something like this to hint that these are the files handled by git.

```suggestion
        git_fragments = run(
```

And shouldn't this be moved inside `if answer_yes` , to hint that this code is only needed if we need to delete the files.

And maybe we can resolve the path to absolute so that we don't have 2 variables.
git should be able to handle absolute paths.